.github/workflows/catalog.deployment.yml

```diff
@@ -0,0 +1,36 @@
+name: Catalog Deployments
+
+on: [workflow_dispatch]
+
+jobs:
+  aws-catalog:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        deployment: [
+          'https://raw.githubusercontent.com/newrelic/demo-catalog/main/catalog/hello.aws.json'
+        ]
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+
+    - name: Write AWS Certificate to File
+      env:
+        AWS_PEM: ${{ secrets.CATALOG_DEPLOY_V3_AWS_US_WEST_1_PEM }}
+      run: |
+        rm -f config/catalog_us_west_1.pem
+        mkdir config
```

Comment:
should be 'configs' to match up with the mount, or change the mount to match up with this.